schema.sql

```diff
@@ -3,4 +3,4 @@
 CREATE TABLE animals(id INT GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
 name VARCHAR(50),
 date_of_birth DATE NOT NULL,escape_attempts INT,neutered VARCHAR(30),
-weight_kg FLOAT);
+weight_kg FLOAT, species TEXT);
```

Comment:
- [ ] Even though this code works, the purpose of this exercise is for you to use the `ALTER` command in this table, so instead of manually editing the table, please use something like `ALTER table animal` and apply the necessary code.